web/config/environment.js

```diff
@@ -32,6 +34,20 @@ module.exports = function (environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+    metricsAdapters: hasGoogleAnalyticsID
+      ? [
+          {
+            name: "GoogleAnalyticsFour",
+            environments: ["development", "production"],
+            config: {
+              id: getEnv("GOOGLE_ANALYTICS_ID"),
```

Comment:
Could we use `GOOGLE_ANALYTICS_TAG_ID` to be consistent with the server config option?